traffic_ops/traffic_ops_golang/role/roles.go

```diff
@@ -122,9 +122,11 @@ func (role *TORole) Create() (error, error, int) {
 	}
 
 	//after we have role ID we can associate the capabilities:
-	userErr, sysErr, errCode = role.createRoleCapabilityAssociations(role.ReqInfo.Tx)
-	if userErr != nil || sysErr != nil {
-		return userErr, sysErr, errCode
+	if role.Capabilities != nil && *role.Capabilities != nil && len(*role.Capabilities) > 0 {
```

Comment:
Nit but this can be simplified to `role.Capabilities != nil && len(*role.Capabilities) > 0`

Reply:
Yeah, so I've heard. Never knew that about Go, makes life much easier